src/Place/ReadModel/Duplicate/CleanPlaceName.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\Duplicate;
+
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Model\ValueObject\Geography\Address;
+
+class CleanPlaceName
+{
+    private const MAX_LENGTH_TITLE = 150;
+    private const NONE = 'none';
+
+    public static function transform(Address $address, string $title): string
+    {
+        if (mb_strtolower($title) === self::NONE) {
+            return '';
+        }
+
+        if (mb_strlen($title) > self::MAX_LENGTH_TITLE) {
+            return '';
+        }
+
+        if (str_contains($title, $address->getStreet()->toString())) {
+            return '';
+        }
+
+        $title = htmlspecialchars($title);
+
+        $title = str_replace([' BE', 'BE ', ' NL', 'NL '], [' Belgium', 'Belgium ', ' Netherlands', 'Netherlands '], $title);
+
+        //Decode the unicode characters
+        return Json::decodeAssociatively('"' . $title . '"');
```

Comment:
There is also the method `Json::decode`.

But it feels weird to use JSON conversion to do Unicode conversion. I guess these are the quirks of PHP. I would expect something like `mb_convert_encoding($unicodeString, 'ASCII//TRANSLIT', 'UTF-8')`

Reply:
I tried, but it gave problems with characters like é, it converted them to ?

Reply:
Thanks for trying!